tools/system_libs.py

```diff
@@ -1271,6 +1271,8 @@ def __init__(self, **kwargs):
     self.memvalidate = kwargs.pop('memvalidate')
     self.verbose = kwargs.pop('verbose')
     self.is_debug = kwargs.pop('is_debug') or self.memvalidate or self.verbose
+    self.thread_safe = kwargs.pop('thread_safe')
```

Comment:
This library already inherits from `MTLibrary` and has the `is_mt` attribute.. is this not the same thing?